src/config/development.tsx

```diff
@@ -184,6 +184,7 @@ const config: EnvConfig = {
 		],
 		regenPools: [],
 		uniswapV2Subgraph:
+			process.env.uniswapV2Subgraph ||
```

Comment:
@mohammadranjbarz We have to use `NEXT_PUBLIC_` before the Name to be accessible on the client too.
also please use UPPERCASE for the name for instance: ‍`NEXT_PUBLIC_SUBGRAPH_UNISWAP_V2`

Reply:
Thanks @MohammadPCh  for your great feedback, done.